Makefile

```diff
@@ -82,7 +86,10 @@ example-kubernetes: container
 example-containers: container
 	examples/containers/run.sh
 
-examples: clean container example-kubernetes example-containers
+example-multi: container
+	examples/containers/multi/run.sh
+
+examples: clean container example-kubernetes example-containers example-multi
```

Comment:
Is there a reason we have a target for starting all examples at the same time? In most cases you'd only want one at a time no?

Reply:
Not sure, this was already there I just rearranged the order to keep it consistent. cc @patrick-stephens

Reply:
I think the original intent was to ensure we can run all the examples as part of CI tests, i.e. there are no failures. I'm happy to remove.